frontend/app/routes/public/renew/$id/child/children/$childId/update-federal-provincial-territorial-benefits.tsx

```diff
@@ -341,22 +341,22 @@ export default function RenewChildUpdateFederalProvincialTerritorialBenefits() {
           </fieldset>
           {editMode ? (
             <div className="mt-8 flex flex-wrap items-center gap-3">
-              <Button variant="primary" id="continue-button" disabled={isSubmitting} data-gc-analytics-customclick="ESDC-EDSC:CDCP Renew Application Form-Child:Save - Access to other dental benefits click">
+              <Button variant="primary" id="continue-button" disabled={isSubmitting} data-gc-analytics-customclick="ESDC-EDSC:CDCP Renew Application Form-Child:Save - Child access to private dental insurance click">
```

Comment:
"Child access to private dental insurance click" should be "Child access to other federal, provincial or territorial dental benefits click" to match the page title.
```suggestion
              <Button variant="primary" id="continue-button" disabled={isSubmitting} data-gc-analytics-customclick="ESDC-EDSC:CDCP Renew Application Form-Child:Save - Child access to other federal, provincial or territorial dental benefits click">
```

The same comment applies for the rest of the attribute values on this page.

Reply:
Fixed page titles